### DISCUSSION TITLE
Drag file from Tauri app to desktop

### DISCUSSION THREAD

**Discussion by channing** [tauri-apps/tauri#4649] [Q&A]
I'm trying to build something like a file manager using Tauri. Is it possible to drag a file (which is on the local machine) displayed on the webview to the desktop or other apps?

Thanks in advance for your help!

---
EDIT: Just found https://github.com/tauri-apps/tauri/issues/2593. Seems not supported natively by Tauri at the moment. Does anyone know a workaround?

**Comment by creeper12356** [comment]
Two years later. 
Does anyone know a workaround to drag file from Tauri app to native file manager window or desktop? :)
